cmd/vochaintest/vochaintest.go

```diff
@@ -123,9 +145,9 @@ func main() {
 			false)
 	case "vtest":
 		mkTreeVoteTest(*host,
-			*oraclePrivKey,
+			oracleKey,
 			*electionType == "encrypted-poll",
-			entityKey,
+			entityKeys[2],
```

Comment:
This looks like a very static approach and hard to understand if the code is not read. Also it is a not consistent since opmode can only be one per execution so we are creating entityKeys that are not being used. Consider the following changes:

- I'd change the naming to `accountKeys`, seems more consistent with the rest of the namings.

- `initaccounts` would take `accountKeys` and `treasurerKey`. Initialize the whole list of account keys provided.

- A test operation such as vtest uses the `accountKeys[0]` only, the rest of test operation also use only `accountKeys[0]`. If `accountKeys` is empty then it requires `treasurerKey` in order to generate a new random key and initialize it.